lib/rpmsg/rpmsg.c

```diff
@@ -264,7 +264,8 @@ void rpmsg_destroy_ept(struct rpmsg_endpoint *ept)
 		return;
 
 	rdev = ept->rdev;
-	(void)rpmsg_send_ns_message(ept, RPMSG_NS_DESTROY);
+	if(ept->name && strncmp(ept->name, "NS", sizeof("NS")))
```

Comment:
instead of compare the name, how about check  the endpoint address

Reply:
yes will change this